doc/ref_cert/lfn/chapters/chapter04.md

```diff
@@ -97,7 +97,71 @@ It covers the test cases against networking management operations.
+| .\*test_security_groups_negative.SecurityGroupsNegativeTestJSON.test_security_group_create_with_duplicate_name            | neutron                            |
+| .\*test_security_groups_negative.SecurityGroupsNegativeTestJSON.test_security_group_create_with_invalid_group_description | https://launchpad.net/bugs/1161411 |
+| .\*test_security_groups_negative.SecurityGroupsNegativeTestJSON.test_security_group_create_with_invalid_group_name        | https://launchpad.net/bugs/1161411 |
+| .\*test_security_groups_negative.SecurityGroupsNegativeTestJSON.test_update_security_group_with_invalid_sg_des            | neutron                            |
+| .\*test_security_groups_negative.SecurityGroupsNegativeTestJSON.test_update_security_group_with_invalid_sg_id             | neutron                            |
+| .\*test_security_groups_negative.SecurityGroupsNegativeTestJSON.test_update_security_group_with_invalid_sg_name           | neutron                            |
+| .\*test_list_server_filters.ListServerFiltersTestJSON.test_list_servers_filtered_by_ip_regex                              | https://launchpad.net/bugs/1540645 |
+| .\*servers.test_virtual_interfaces                                                                                        | nova-network                       |
+| .\*compute.test_virtual_interfaces_negative                                                                               | nova-network                       |
+| .\*compute.test_networks                                                                                                  | nova-network                       |
+| .\*test_attach_volume.AttachVolumeMultiAttach                                                                             | volume_multiattach                 |
+| .\*test_volume_boot_pattern.TestVolumeBootPattern.test_boot_server_from_encrypted_volume_luks                             | attach_encrypted_volume            |
+| .\*test_volume_swap                                                                                                       | swap_volume                        |
+| .\*test_encrypted_cinder_volumes                                                                                          | attach_encrypted_volume            |
+| .\*test_stamp_pattern.TestStampPattern.test_stamp_pattern                                                                 | https://launchpad.net/bugs/1664793 |
+| .\*test_volume_migrate_attached                                                                                           | https://launchpad.net/bugs/1664793 |
+| .\*test_minbw_allocation_placement                                                                                        | microversion                       |
+| .\*test_network_advanced_server_ops.TestNetworkAdvancedServerOps.test_server_connectivity_cold_migration_revert           | cold_migration                     |
```

Comment:
Why skip cold/live migration when they are mandatory in RA?

Reply:
As far as I understand, Ceph is mandatory in RA/RI which force not executing block and volume-backed live migration  (here a few of them ask for iscsi).

cold migration is well tested, possibly a false reason (checking git history) 
https://git.opnfv.org/functest/tree/docker/smoke-cntt/tempest_conf.yaml

Reply:
Then we should fix the requirements. If there are conflicting requirements or missing feature it should be listed in RA. User will be using this to test their own NFVI and they might have support for it.

Reply:
@deebhatia 
Yes. Thank you I know the process here and I have deeply contributed in both.
For my understanding the support is live migration as proposed and the testing part differs between the backend. As ceph is selected, the testing part becomes simple.

Would you mind approving the PR (and the others in the same topic), it's filling the gap between what's implemented and what's documented. I don't see any true reason on your side in this PR chain asking for an update.